js/app.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
+/**
+ * Construct and fill Product Array
+ */
+var urls = ['../img/bag.jpg', '../img/banana.jpg', '../img/bathroom.jpg', '../img/boots.jpg', '../img/breakfast.jpg', '../img/bubblegum.jpg', '../img/chair.jpg', '../img/cthulhu.jpg', '../img/dog-duck.jpg', '../img/dragon.jpg', '../img/pen.jpg', '../img/pet-sweep.jpg', '../img/scissors.jpg', '../img/shark.jpg', '../img/sweep.png', '../img/tauntaun.jpg', '../img/unicorn.jpg', '../img/usb.gif', '../img/water-can.jpg', '../img/wine-glass.jpg'];
 
-console.log('JS linked');
\ No newline at end of file
+var allProducts = []; //Holds all products
+var displayedProducts = []; //Holds products that have been displayed
+var currentDisplay = []; //Holds products that are currently displayed
+for(var i = 0; i < urls.length; i++ ) {
+  allProducts.push(new Product(urls[i]));
+}
+var ITERATIONS = 25; //Number of voting rounds
+var NUMBER_DISPLAYED = 3; //How many images are displayed per voting round
+var iterationCounter = 0;
+
+var firstImage = document.getElementById('firstImage'); //HTML location of first image
+var secondImage = document.getElementById('secondImage'); //HTML location of second image
+var thirdImage = document.getElementById('thirdImage'); //HTML location of third image
+
+var formElement = document.getElementById('chooseProduct'); //HTML location of form
+formElement.addEventListener('submit', handleVote);
+
+
+/**
+ * Constructor for listed items.
+ * @param {string} url Location of product image in img directory
+ */
+function Product(url) {
+  this.url = url;
+  this.votes = 0;
+  this.displays = 0;
+}
+
+function selectImages() {
+  for(var i = 0; i < NUMBER_DISPLAYED; i++) {
+    if(allProducts.length === 0) {
+      allProducts = allProducts.concat(displayedProducts);
+    }
+    var randomNum = Math.floor(Math.random() * allProducts.length);
+    currentDisplay.push(allProducts[randomNum]);
+    allProducts.splice(randomNum, 1);
```

Comment:
Something that's odd to me is that this array is called `allProducts`, but you remove things from that array as your code runs. I'd consider calling this `productsToBeDisplayed` or similar.